environment.yml

```diff
@@ -70,7 +70,7 @@ dependencies:
   - xlsxwriter
   - zarr
   # Python packages needed for unit testing
-  - flake8
+  - flake8 ==5.0.4
```

Comment:
@bouweandela this breaks compatibility with ESMValCore! We pin it to `flake8 >= 7` there

Reply:
nevermind, it doesn't, not at env creation and install level, at least, all good :+1: